lib/fathead/mdn_css/fetch.pl

```diff
@@ -67,7 +67,47 @@
 my $current_active_connections = 0;
 my $maximum_active_connections = 4;
 
-my @keyword_urls = keys %urls;
+#save the urls with fragments to a text file called
+#fragments.txt so that parse.pl can use this information
+#to extract extra information about the fragments
+my @keyword_urls        = map  { Mojo::URL->new($_) } keys %urls;
+my @urls_with_fragments = grep { $_->fragment } @keyword_urls;
+if (@urls_with_fragments) {
+    open( my $fh, '>:encoding(UTF-8)', catfile 'download', 'fragments.txt' )
+      or die $!;
+
+=begin
+    For fragment in urls past this path transform-function we can get
+    the link for the actual fragment in the following way.
+    Take for example this fragment #matrix()
+    In order to get the actual link, we take the fragment, matrix(), remove
+    the brackets, and append it to the original path so that we get the link
+    https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix
+=cut
+
+    for my $url_with_fragment (@urls_with_fragments) {
+        if ( $url_with_fragment =~ qr/transform-function/ ) {
```

Comment:
I realized that for the fragments under `transform-functions`, there are actual links to specific pages that have the format like other CSS pages and hence they can be parsed like the other normal pages. Therefore, instead of writing the fragments to file, I made the fetch.pl script build [correct links from fragments](https://github.com/duckduckgo/zeroclickinfo-fathead/pull/284/files/70e19564e1326ce0b8a77c2676995b22bd923b30#r73949668) and dowload the actual pages.

For instance, [#matrix()](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function#matrix()] becomes this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix) which when downloaded can be parsed by parse.pl because it appears like the other normal pages.

Other fragments are written to file because they require special parsing as can be seen, for instance [from here](https://github.com/duckduckgo/zeroclickinfo-fathead/pull/284/commits/124583b7da4627767715d3e3b8476a0578a07c12)